"make" in book/ failure starting at 3b623a1

As of commit 3b623a1 (associated with the changes in #6724), a `make` in the `book/` directory has started failing, which blocks redeployment of docs updates.

```
$ git rev-parse HEAD
3b623a11660b4e7fdd494221c37e2269ca7dceec

$ cd book && make
GOOS=js GOARCH=wasm go build -o src/super.wasm super-example/main.go
# command-line-arguments
super-example/main.go:90:34: cannot use query (variable of interface type "github.com/brimdata/super/runtime".Query) as sbuf.Puller value in argument to sbuf.CopyPuller: "github.com/brimdata/super/runtime".Query does not implement sbuf.Puller (wrong type for method Pull)
		have Pull(bool) (vector.Any, error)
		want Pull(bool) (sbuf.Batch, error)
make: *** [all] Error 1
```

--- a/book/super-example/main.go
+++ b/book/super-example/main.go
@@ -16,9 +16,9 @@ import (
 	"github.com/brimdata/super/compiler/parser"
 	"github.com/brimdata/super/pkg/storage"
 	"github.com/brimdata/super/runtime"
-	"github.com/brimdata/super/sbuf"
 	"github.com/brimdata/super/sio"
 	"github.com/brimdata/super/sio/anyio"
+	"github.com/brimdata/super/vector/vio"
 	"github.com/teamortix/golang-wasm/wasm"
 )
 
@@ -87,7 +87,7 @@ func run(opts opts) wasm.Promise {
 			return "", err
 		}
 		defer query.Pull(true)
-		if err := sbuf.CopyPuller(zwc, query); err != nil {
+		if err := vio.Copy(zwc, query); err != nil {
 			return "", err
 		}
 		if err := zwc.Close(); err != nil {

